README.md

```diff
@@ -1,17 +1,16 @@
 # gluster-kubernetes
 
-## Dynamic, persistent storage for Kubernetes with GlusterFS
+## GlusterFS + heketi on Kubernetes
 
-This is the home of the **gluster-kubernetes** project,
-a new project which brings dynamically provisioned persistent
-storage volumes to kubernetes, using a hyperconverged Gluster
-deployment scheme.
+**gluster-kubernetes** is a project to provide Kubernetes administrators a
+mechanism for deploying GlusterFS and heketi onto their clusters.
```

Comment:
Same point of criticism as above. Exchanging the higher level mission with the lower level technical way to achieve it.